src/components/CopyToClipBoard/CopyToClipBoard.test.tsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
-import { mount, ReactWrapper } from 'enzyme';
 
 import { copyToClipBoardUtility } from '../../utils/cli-utils';
+import { mount } from '../../utils/test-enzyme';
 
 import CopyToClipBoard from './CopyToClipBoard';
 import { CopyIcon } from './styles';
 
 jest.mock('../../utils/cli-utils');
 
 describe('<CopyToClipBoard /> component', () => {
-  let wrapper: ReactWrapper;
```

Comment:
Any reason why you removed the type here? This now errors again in strict mode, it was already covered :(

Reply:
It might be a mistake, I restored them.

Reply:
Restored

Reply:
Sorry @tmkn I was getting errors due to these types, I removed it for testing purposes and forgot to undo my changes ... good catch!

Reply:
@priscilawebdev no worries, I was sure it wasn't intentional, that's what code reviews are for :)